app/pages/multisig/updates/AddIdentityProvider/AddIdentityProviderView.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { AddIdentityProvider } from '~/utils/types';
+import { hashSha256 } from '~/utils/serializationHelpers';
+import { fieldDisplays } from './CreateAddIdentityProvider';
+import { noOp } from '~/utils/basicHelpers';
+
+interface Props {
+    addIdentityProvider: AddIdentityProvider;
+}
+
+/**
+ * Displays an overview of an "Add Identity provider" transaction payload.
+ */
+export default function AddIdentityProviderView({
+    addIdentityProvider,
+}: Props) {
+    const [hash, setHash] = useState<string>('');
+
+    useEffect(() => {
+        hashSha256(Buffer.from(addIdentityProvider.ipVerifyKey, 'hex'))
```

Comment:
This looks like a `node` Buffer and not the imported one from `import { Buffer } from 'buffer/';`.

Reply:
We could also import the hash function from `hash.js` now and make `hashSha256` synchronous again.

Reply:
We can use the inbuilt hash synchronously again, from the preload script, I have updated the function to do this.